src/test/resources/org/broadinstitute/hellbender/tools/copynumber/gcnv-postprocess/shard_0-calls/SAMPLE_0/mu_denoised_copy_ratio_t.tsv

```diff
@@ -0,0 +1,225 @@
+@RG	ID:GATKCopyNumber	SM:SAMPLE_000
```

Comment:
Wait, so are these files supposed to have `dtype` and `shape` header lines?  I thought all `mu_` and `std_` files would?

Reply:
Fixed this, some calls to `write_ndarray_to_tsv` overriden `write_shape_info`